runtime/lua/vim/lsp/util.lua

```diff
@@ -760,9 +817,9 @@ function M.apply_workspace_edit(workspace_edit)
     for idx, change in ipairs(workspace_edit.documentChanges) do
       if change.kind == "rename" then
         M.rename(
-          vim.uri_to_fname(change.oldUri),
```

Comment:
Could you revert the unrelated formatting changes?

Reply:
Oops, format-on-save getting me once again :sweat_smile: